### ISSUE TITLE
Investigations with Gitpod?

### ISSUE THREAD

**Issue by jmini** [docToolchain/docToolchain#679]
Did somebody investigated how to use docToolchain with https://www.gitpod.io/?

Not sure if this is possible, but we should configure it so that:

* The editor they provide (similar to VSCode) has some of the interesting plugins (the [AsciiDoc](https://open-vsx.org/extension/asciidoctor/asciidoctor-vscode) and maybe the [Diagrams.net](https://open-vsx.org/extension/hediet/vscode-drawio))
* It is possible to run the `./dtcw` locally with everything needed inside the pod.
* A http port is correctly configured so that [previewSite](http://doctoolchain.org/docToolchain/v2.0.x/015_tasks/03_task_previewSite.html) is working as expected

This should be for this repo, but also something that other docToolchain-based reposity can copy.

**Comment by rdmueller** [comment]
I love gitpod! I guess we should create a tutorial which explains how to use dtcw inside a gitpod container of your choice...

**Comment by rdmueller** [comment]
we should also add a .devcontainer definition for VS Code

**Comment by jmini** [comment]
Isn't it more a `.gitpod.yml` file?

See https://www.gitpod.io/docs/configure

I did not took the time to look into it yet. Still on my TODO list

**Comment by rdmueller** [comment]
yes, it is a `.gitpod.yml` file for gitpod, but it is a `.devcontainer` folder for VS Code: https://code.visualstudio.com/docs/remote/create-dev-container#:~:text=A%20devcontainer.json%20file%20in%20your%20project%20tells%20VS,or%20runtimes%20needed%20for%20working%20with%20a%20codebase.

let's add both!
